app/views/candidates/sessions/new.html.erb

```diff
@@ -16,7 +16,7 @@
       <%= f.email_field :email, autocomplete: 'on', class: 'govuk-input govuk-input--width-30' %>
       <%= f.text_field :firstname, autocomplete: 'given-name', class: 'govuk-input govuk-input--width-20' %>
       <%= f.text_field :lastname, autocomplete: 'family-name', class: 'govuk-input govuk-input--width-20' %>
-      <%= f.date_field :date_of_birth %>
+      <%= f.date_field :date_of_birth, autocomplete: 'bday', heading: true %>
```

Comment:
Afaik the `bday` attr won't work with the current formbuilder because the single `date_field` is split up into three inputs, they'd need separate attributes `bday-day`, `bday-month` and `bday-year`.

Reply:
Ok, this was copied from rich's original work on adding date of birth. I'll remove that for now